tests/snapshot/main.rs

```diff
@@ -58,6 +58,40 @@ fn check_error_snapshots(file: &str) {
     });
 }
 
+#[test_resources("tests/snapshot/inputs/docs/*.ncl")]
+fn check_doc_stdout_snapshots(file: &str) {
+    let file = TestFile::from_project_path(file);
+
+    let snapshot = NickelInvocation::new()
+        .subcommand("doc")
+        .file(&file)
+        .args(["--stdout"])
+        .snapshot_stdout();
+
+    insta::assert_snapshot!(file.prefixed_test_name("doc_stdout"), snapshot);
+}
+
+#[test_resources("tests/snapshot/inputs/docs/*.ncl")]
+fn check_doc_stderr_snapshots(file: &str) {
+    let file = TestFile::from_project_path(file);
+
+    let snapshot = NickelInvocation::new()
+        .subcommand("doc")
+        .file(&file)
+        .args(["--stdout"])
+        .snapshot_stderr();
+
+    insta::with_settings!({filters => vec![
```

Comment:
Don't we do something like this already elsewhere in the test framework? Just to know if we could factor this out (e.g. in `utilities`)